### DISCUSSION TITLE
A few suggestions: Fullscreen, TV Mode, playback of Theme Videos and Theme songs

### DISCUSSION THREAD

**Discussion by Thursty24** [prayag17/Blink#454] [Ideas]
Hi There, I have a few ideas that I think will be great!!!!

Ideally I want to build a linux media player for my Home Theatre, currently I'm working on using Jellyfin Media Player (Not really a fan), so I want to give a few ideas that will help make Blink into the best it can be:

1. Enable fullscreen mode
2. Jellyfin has a TV mode option that allows keyboard or remote navigation, would love to see this on here too.
3. Theme Video Backdrops, some of the movies on many servers has backdrop videos, would be great to see Blink show those too.
4. Theme songs, same as above, I think this is a neat feature.
5. Chapter Navigation in the Movie info and player screen.
6. Option to change audio while watching the movie

**Comment by prayag17** [comment]
> Enable fullscreen mode

This issue has been mentioned before and I will push a commit fixing this "bug" 

> Jellyfin has a TV mode option that allows keyboard or remote navigation, would love to see this on here too.

Blink already supports keyboard navigation with the `tab` key but it does not support movement like jellyfin-web and I don't plan on adding that in v1. This can also be paired with controller navigation support which is outside the scope of this project

> Theme Video Backdrops, some of the movies on many servers has backdrop videos, would be great to see Blink show those too.

This has been suggested in past but don't like the idea of adding a theme-video-backdrop since the current backdrop has a heavily applied blur filter on it which adds to the look of this application, I dont plan on losing it in favor of a backdrop video and personally I feel theme videos to be a bit distracting while browsing.

> Theme songs, same as above, I think this is a neat feature.

That seems like a good suggestion, I will look into it in alpha 4

> Chapter Navigation in the Movie info and player screen.

This feature is already planned for alpha 3 
You can track it at #457

> Option to change audio while watching the movie

This feature is also planned but I have not found a way to implement this yet...

**Reply by prayag17** [reply]
@Thursty24 a follow up to this discussion, I just pushed a commit which allows changing audio track from video player. It not perfect but I am working on improving its UX

**Reply by Thursty24** [reply]
That's perfect! Thank you

Please can I have more info on the controller navigation? What controller are we talking?

**Reply by Thursty24** [reply]
I think Blink has so much potential to become the ultimate jellyfin client especially for builds that people like me want to integrate to its own media player unit.

**Reply by prayag17** [reply]
> That's perfect! Thank you
> 
> Please can I have more info on the controller navigation? What controller are we talking?

I am talking about gamepads and tv remotes. it is not yet implemented
